CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+4. Click the edit (pencil) button
+
+From there, the GitHub UI will walk you through the rest by creating a [fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) and a branch to commit your changes to.
+After you have made changes to your branch, the goal is to open a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) for your changes to be incorporated.
+
+A pull request represents the work you want to be reviewed, approved, and merged into the `main` branch of the MDN repository.
+See the [Creating a pull request](#creating-a-pull-request) for more details on creating and handling pull requests successfully.
+
+If you're not certain of the changes that you want to make, [get in touch with us](/en-US/docs/MDN/Community/Communication_channels)!
+
+> **Note:** You can click the **View the source on GitHub** or **Edit the page on GitHub** link at the bottom of an MDN page to jump directly to the page source on GitHub.
+
+### Forking and cloning the repository
+
+If you want to make changes to more than one file, the GitHub UI is not very efficient because you have to make separate pull requests for each file you want to change.
+Instead of using the GitHub UI, you need to use `git` or a client like [GitHub Desktop](https://docs.github.com/en/get-started/using-github/github-desktop) or [GitHub CLI](https://docs.github.com/en/github-cli/github-cli/about-github-cli).
+
+#### Set up `mdn/content` repo
+
+First, you will need to set up the `mdn/content` repo locally. If you do not plan to contribute to upstream content, you will not need to fork it as described in its [Contributing guide](https://github.com/mdn/content/blob/main/CONTRIBUTING.md#forking-and-cloning-the-repository); all you need to do is clone it by running the following command:
+
+```bash
+git clone git@github.com:mdn/content.git
+```
+
+After the repository has been cloned, follow the [steps to prepare the project](https://github.com/mdn/content/blob/main/CONTRIBUTING.md#preparing-the-project).
+
+#### Set up `mdn/translated-content` repo
+
+Once that is done, you will also need to fork and clone this repository by following the [upstream Contributing guide](https://github.com/mdn/content/blob/main/CONTRIBUTING.md#forking-and-cloning-the-repository), replacing `content` with `translated-content`.
+
+#### Link translated content
+
+1. Navigate to your local clone of the content repository fork:
+
+   ```bash
+   cd /path/to/content
+   ```
+
+2. Next, define an environment variable in a `.env` file called `CONTENT_TRANSLATED_ROOT` containing the path to the _translated-content_ repo’s `files` directory:
+
+   ```bash
+   # macOS/Linux
+   echo CONTENT_TRANSLATED_ROOT=/path/to/translated-content/files >> .env
+   ```
+
+   ```batch
+   # Windows
+   echo CONTENT_TRANSLATED_ROOT=\path\to\translated-content\files > .env
+   ```
```

Comment:
Maybe we don't need to specify the path for Windows, both slashes and backslashes work on Windows.

Reply:
A part of the reason I'm specifying the Windows command is because Windows uses a single angle bracket for piping rather than two!

Reply:
We could using single angle bracket or double angle brackets on Windows. The behavor is the same as terminal on Linux/MacOS. I've tested this on cmd and powershell.

![image](https://github.com/mdn/translated-content/assets/15844309/ee65e648-81e3-4142-be3c-b82e56c5951e)

Reply:
Ah, good to know!  (I don't use Windows so I only have the internet to tell me how it works.  :P)